Sources/RInAppMessaging/Extensions/UILabel+IAM.swift

```diff
@@ -1,3 +1,5 @@
+import UIKit
```

Comment:
`import UIKit.UILabel` ?

Reply:
Then I'd have to add explicit import for CGFloat and NSMutableParagraphStyle...